app/__init__.py

```diff
@@ -47,10 +48,16 @@ def __call__(self, environ, start_response):
         return self.app(environ, start_response)
 
 
-def create_app():  # noqa: C901  pylint: disable=too-complex
+def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-complex
     application = Flask(__name__, static_url_path='/s', static_folder='../static')
     application.config.from_object(settings)
 
+    secrets = yaml.safe_load(open(os.getenv('EQ_SECRETS')))
```

Comment:
Yes that makes total sense. Longer term I was thinking that we'd move all config loading out of create_app to another place (e.g. application.py)? So create_app would only be concerned with putting the app together based off the config object|dict that's passed to it. That will help with [simplifying create_app](https://trello.com/c/90vVTmqt/425-look-at-reducing-the-complexity-of-create-app-app-init-py).

I'm probably jumping the gun a bit here though, happy with it as is.